packages/app-wallet-interface/source/index.html.md

```diff
@@ -132,6 +132,29 @@ Returns the signing address(es) for the current domain.
 }
 ```
 
+## GetEthereumSelectedAddress
+
+Returns the ethereum address selected in metamask. Typically used as the `destination`.
```

Comment:
We use `destination` in this doc, but the `rps` uses `outcomeAddress`. Should we change?